sormas-ui/src/main/java/de/symeda/sormas/ui/importer/DataImporter.java

```diff
@@ -135,11 +135,14 @@ public abstract class DataImporter {
 
 	private final EnumCaptionCache enumCaptionCache;
 
+	private final boolean singleColumnImport;
+
 	public DataImporter(File inputFile, boolean hasEntityClassRow, UserDto currentUser, ValueSeparator csvSeparator) throws IOException {
 		this.inputFile = inputFile;
 		this.hasEntityClassRow = hasEntityClassRow;
 		this.currentUser = currentUser;
 		this.enumCaptionCache = new EnumCaptionCache(currentUser.getLanguage());
+		this.singleColumnImport = false;
```

Comment:
Please call the constructor below instead of duplicating the whole logic

Reply:
Done